src/shared/Microsoft.Git.CredentialManager/EnumerableExtensions.cs

```diff
@@ -25,5 +25,28 @@ public static IEnumerable<TSource> ConcatMany<TSource>(this IEnumerable<TSource>
 
             return result;
         }
+
+        /// <summary>
+        /// Creates a <see cref="HashSet{TSource}"/> from an <see cref="IEnumerable{TSource}"/>.
+        /// </summary>
+        /// <param name="sequence">Sequence to construct a <see cref="HashSet{TSource}"/> with.</param>
+        /// <typeparam name="TSource">Type of the sequence elements.</typeparam>
+        /// <returns><see cref="HashSet{TSource}"/> containing unique elements from the sequence.</returns>
+        public static HashSet<TSource> ToHashSet<TSource>(this IEnumerable<TSource> sequence)
```

Comment:
Are these used?

Reply:
They used to be used in an earlier revision.. removing..